Cargo.lock

```diff
@@ -41,24 +41,15 @@ dependencies = [
  "xdg",
 ]
 
-[[package]]
```

Comment:
Could you give a summary about the dependency tree changes?

Reply:
Update some dependencies to pass the compile